Jenkinsfile

```diff
@@ -48,7 +48,10 @@ runTheBuilds.runDevToolsProject(
     if (shouldDeploy) {
       data['dtrImage'].push()
       data['dtrImage'].deploy(
-        '8000', '-v jenkins-nodes:/jenkins_nodes', env.CONTAINER_ARGS)
+        '8000',
+        '-v jenkins-nodes:/jenkins_nodes',
+        "${env.JENKINS_URL} /jenkins_nodes/output.json",
```

Comment:
how will this work with staging environments? would the staging server deploy its own scanner?

Reply:
Yes, that is the theoretical idea. We'd need to mofidy the `DTRImage` class though to provide the ability to override `DOCKER_HOST`, which is hardcoded to `reed`. Or perhaps overriding the container name is enough, but we'd need to do something.